src/foundation/Icon/Svg/Pin.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M15.4978 2.36382C15.2138 2.26398 14.9111 2.22917 14.6119 2.26197C14.3127 2.29477 14.0246 2.39434 13.7691 2.55333C13.5135 2.71232 13.2968 2.92686 13.1351 3.18075L9.91993 8.23364C8.31391 7.97553 6.66148 8.27865 5.2456 9.10163L5.24297 9.10317C4.97782 9.25874 4.75189 9.47303 4.58254 9.72959C4.41319 9.98615 4.30493 10.2781 4.2661 10.5831C4.22727 10.888 4.2589 11.1978 4.35855 11.4886C4.45813 11.7792 4.62344 12.0436 4.84092 12.2606L7.75992 15.1795L2.46968 20.4697C2.17678 20.7625 2.17677 21.2374 2.46966 21.5303C2.76256 21.8232 3.23743 21.8232 3.53033 21.5303L8.82059 16.2402L11.7401 19.1597C11.9571 19.3771 12.221 19.5419 12.5115 19.6415C12.8023 19.7411 13.1121 19.7727 13.4171 19.7339C13.722 19.6951 14.014 19.5868 14.2706 19.4175C14.5271 19.2481 14.7414 19.0222 14.897 18.7571L14.8985 18.7544C15.7215 17.3386 16.0247 15.6861 15.7666 14.0801L20.8186 10.8651C21.0726 10.7036 21.287 10.4871 21.4462 10.2316C21.6053 9.97605 21.705 9.68806 21.7379 9.38885C21.7709 9.08965 21.7362 8.78687 21.6364 8.50286C21.5367 8.21886 21.3743 7.9607 21.1615 7.74778L16.2524 2.83881C16.0396 2.62598 15.7817 2.46366 15.4978 2.36382ZM14.7753 3.75304C14.8513 3.74471 14.9281 3.75355 15.0002 3.77889C15.0723 3.80423 15.1377 3.84546 15.1918 3.89948L20.1008 8.80845C20.1548 8.86246 20.1959 8.92788 20.2212 8.99988C20.2465 9.07197 20.2553 9.14882 20.2469 9.22475C20.2386 9.30069 20.2133 9.37379 20.1729 9.43864C20.1325 9.50349 20.0779 9.55855 20.0134 9.59956L14.528 13.0904C14.2577 13.2624 14.1265 13.5871 14.2015 13.8986C14.5367 15.2916 14.3222 16.7601 13.6026 17.999C13.5632 18.0659 13.509 18.1228 13.4443 18.1656C13.3791 18.2086 13.305 18.2361 13.2276 18.2459C13.1502 18.2558 13.0716 18.2477 12.9978 18.2224C12.9239 18.1972 12.8569 18.1553 12.8018 18.1L5.90077 11.1991L5.89998 11.1983C5.84473 11.1432 5.80284 11.0762 5.77755 11.0024C5.75226 10.9286 5.74423 10.8499 5.75409 10.7726C5.76394 10.6952 5.79142 10.621 5.8344 10.5559C5.87716 10.4912 5.93413 10.437 6.00097 10.3976C7.23989 9.67798 8.7085 9.46349 10.1015 9.79872C10.413 9.87368 10.7378 9.74248 10.9098 9.47217L14.4006 3.98601C14.4417 3.92165 14.4966 3.86731 14.5614 3.82699C14.6263 3.78664 14.6994 3.76136 14.7753 3.75304Z" fill="#2E3232"/>
```

Comment:
You can remove the `fill`, the color is passed from the component

Reply:
Sure, done 👍